railgun/cmd/rootcmd/env.go

```diff
@@ -22,7 +29,10 @@ func bindEnvVars(cmd *cobra.Command, _ []string) {
 		}
 
 		if envValue, envSet := os.LookupEnv(envKey); envSet {
-			cmd.Flags().Set(f.Name, envValue)
+			if err := f.Value.Set(envValue); err != nil {
+				fmt.Fprintf(os.Stderr, "ERROR: could not set %s value from environment to %s: %s\n", envKey, envValue, err)
```

Comment:
This log print may be unnecessary after this function returns an error (as suggested in the comment on the line below), but I think that we should use the logger here and in other situations like this.

Printing directly to stderr adds a new unrelated channel of reporting, disregards the logging configuration (level, format) and structure.

Reply:
gone with 1455d231067c3007b6dd26b3563f70ec5238912d  :+1: